loader/vk_loader_platform.h

```diff
@@ -372,7 +372,13 @@ static inline const wchar_t *LoaderPnpILayerRegistryWide() {
 
 // File IO
 static bool loader_platform_file_exists(const char *path) {
-    if ((_access(path, 0)) == -1)
+    int path_utf16_size = MultiByteToWideChar(CP_UTF8, 0, path, -1, NULL, 0);
+    if (path_utf16_size <= 0) {
+        return false;
+    }
+    wchar_t *path_utf16 = (wchar_t *)_alloca(path_utf16_size * sizeof(wchar_t));
```

Comment:
`loader_stack_alloc` should be used here, even though it would mean making `allocation.c` transitively included in basically all other loader files.

Reply:
There is a little problem with `allocation.h`. I tried to use `allocation.h`, but loader_stack_alloc was missing because, `allocation.h` has `#include "vk_loader_platform.h"` (via `loader_common.h`).
I think that moving `loader_stack_alloc` before include section will solve this. Do you have better idea?